resources/core/charts/console/values.yaml

```diff
@@ -34,3 +34,4 @@ cluster:
   faviconUrl: 'favicon.ico'
   # Provide custom title for the UI application that will be shown in the header component
   headerTitle: ''
+  disabledNavigationNodes: []
```

Comment:
I added string values to the values.yam here as follows:
```
disabledNavigationNodes: [
    "namespace.operation.secrets",
    "settings",
    "foo.bar"
  ]
```

which generated the following snippet in the config-map:
```
disabledNavigationNodes: '[namespace.operation.secrets settings foo.bar]'
```

This was not properly interpreted by console and the nodes were not hidden

Reply:
Fix proposed : https://github.com/kyma-project/console/pull/995